biom/__init__.py

```diff
@@ -73,5 +73,10 @@
                        {'environment': 'A'}], input_is_dense=True)
 
 
+def concat(tables, *args, **kwargs):
+    """Wrapper for biom.Table.concat which requires a table instance"""
```

Comment:
Can you add the little bit of numpy doc here? Seems like a very handy function that would be good to have well documented.

Reply:
Ya sounds good